docs/sdn_controller.md

```diff
@@ -24,6 +24,9 @@ In the network creation view:
 - Create the network
 - Have fun! ☺
 
+***NB:***
+- All hosts in a private network must be able to reach the other hosts' management interface.
```

Comment:
Is it clear enough?

Should we link to XenServer documentation on how to check/change this?

Reply:
I'll look for such a doc.

Reply:
XenServer? You mean Citrix Hypervisor right :laughing: 
IDK if there is a documentation on such basic thing. A synchronization need to be done for XAPI DB on each node, and this is going through the management interface.

Reply:
I found a doc explaining what's the management interface, where to find it and how to change it with the emergency reset.

I think it's good to refer it. :)

Reply:
Well, this link doesn't explain what's a management interface.

There's just [a note](https://docs.citrix.com/en-us/citrix-hypervisor/networking.html) in Citrix doc:

> The term ‘management interface’ is used to indicate the IP-enabled NIC that carries the management traffic. The term ‘secondary interface’ is used to indicate an IP-enabled NIC configured for storage traffic.

Reply:
It might be enough to put that in a "note" too.

Reply:
Yes I can add the note in my doc but is it the right place to document it?
Management interface is not only used by the SDN Controller.

Reply:
That's why I'm not sure it makes sense to talk about this here.

Reply:
IMHO, if we talk about management interface here, we should link to a documentation with more info or we risk getting questions from users.

Reply:
If we got a question on SDN controller about what's a management interface, there is something wrong somewhere. In the future, yes, we shall probably create a "XCP-ng architecture" page to explain concepts and probably link it from here if needed.

Reply:
I don't the downside of adding this link here even if that's not perfect :thinking: 

At least it helps users understand what it is and where to change it and avoid questions of how to change it from XO…

Reply:
The thing is: if a host can't reach another host's management interface, the tunnels won't work and the network will be unusable.
(for instance it's not possible to create a cross-pool private networks if the 2 pools management interface are isolated from each other)

So I think the NB should be added to the doc so that it is documented.

As for the 'What is a management interface' ? I don't know. I think it's harmless to add it but I'm ok either way.

Reply:
The link needs to be relevant then. Maybe it's not clear here: it's not a question to change the management interface. At all. It's a question to have management interface reachable from each other, which is a really basic thing in a pool, but less common between pools.

Reply:
Makes sense :+1:

Reply:
I guess I can remove the link and just add a short note explaining what's the management interface.
And once we have a XCP-ng doc I'll refer to it?

What do you think?

Reply:
LGTM! :+1: